### PR TITLE
Refactor/pet object

### PR DESCRIPTION
**Pull request by judy0ye**
What I did: Add in more data for pet object and refactored names to match incoming branch's changed property names

Testing Notes/Extras: 

Did this break anything?
- [x]  No
- [ ]  Yes

Type of change
- [x]  New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [ ]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [ ]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next? Styling PetDetails on FE

### CODE REVIEW

**Review by paulina-isabel — APPROVED** [review]
Looking good, the additional pets were a nice move.

**Review by apete12 — APPROVED** [review]
Everything looks great, thanks Judy!

File: server.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ app.locals.pets = [
     id: 1, 
     name: 'Oreo', 
     nickname: 'Oreo', 
-    age: '2.5', 
+    age: 2.5,
```
Comment (nicktu12) [inline server.js:16]: 💪🏻

**Review by aliceabarca — APPROVED** [review]
great job judy!

**Pull request merged**